content/ConnectingToDatabase/ConnectingToDatabase.facts.md

```diff
@@ -0,0 +1,56 @@
+### Facts for Connecting to Database lesson:
+
+`PDO_PGSQL` driver is a tool of `PHP Data Objects (PDO) Interface` to allow connection between PHP and PostgreSQL database.
+
+`PDO` class represents a connection among PHP and a database server.
```

Comment:
Is it possible to replace the word `among` with a different word that is closely related to `between`. Thanks.